### PR TITLE
Modify doc description "~impossible"

### PR DESCRIPTION
**Pull request by lixiaobing1**
File "docs\proposals\cluster-deployment.md", line 126, "Each node can be deployed separately and the implementation should make it ~impossible to change this assumption.", here "~impossible" should be "impossible".

### CODE REVIEW

**Comment by bgrant0607** [comment]
ok to test

**Comment by lixiaobing10051267** [comment]
@bgrant0607 LGTM?

**Comment by bgrant0607** [comment]
@lixiaobing10051267 I neither wrote nor reviewed the original proposal, but I doubt the `~` was accidental. I assigned the PR to people who should know what the original intent was.

File: docs/proposals/cluster-deployment.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ controller works - user can create RC object and RC manager can create pods base
 ## Step 4: Deploy nodes
 
 **Objective**: Start kubelet on all nodes and configure kubernetes network.
-Each node can be deployed separately and the implementation should make it ~impossible to change this assumption.
```
Comment (fgrzadkowski) [inline docs/proposals/cluster-deployment.md]: It was intentional. I mean "almost impossible" or "very hard".

**Comment by lixiaobing10051267** [comment]
ok:)

**Pull request closed without merging**